src/components/NavBar/NavbarDesktop.jsx

```diff
@@ -61,9 +60,7 @@ const NavbarDesktop = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-            PASS
-          </Typography>
+        <img src='src/assets/pass-logo.png' alt='logo' className='navbar-logo'/>
```

Comment:
It looks like a unit test is failing because you removed the `aria-label` attribute. The test is looking for that label to figure out where the logo is, and make sure the navbar is rendering properly. Could you add that attribute to this component?